app/assets/javascripts/app/error_tracking/error_tracking.js

```diff
@@ -0,0 +1,32 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('council.error_tracking', [])
+    .run(errorTracking);
```

Comment:
I was thinking it would make sense to have a controlled initialise because you wouldn't want to run this in development. But since this is controller by the backend, :+1: